Client/Frontend/Browser/Onboarding/Welcome/ReleaseNotesView.swift

```diff
@@ -0,0 +1,94 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveShared
+
+struct ReleaseNotesView: View {
+    @Environment(\.colorScheme) var colorScheme: ColorScheme
```

Comment:
Make private

Reply:
Sorry for inconvenience. Product decided to remove Release Notes from first version. Possibly the design will change also.
The View and present method is deleted.

Removed at https://github.com/brave/brave-ios/pull/4508/commits/79cef7b33427ec27bc534f34e6076574bd198213